xbmc/cores/RetroPlayer/shaders/ShaderPreset.cpp

```diff
@@ -93,44 +91,31 @@ void CShaderPreset::SetVideoSize(unsigned int videoWidth, unsigned int videoHeig
 }
 
 bool CShaderPreset::SetShaderPreset(const std::string& shaderPresetPath)
-{
-  m_presetPath = shaderPresetPath;
-  m_bPresetNeedsUpdate = true;
-  return Update();
-}
-
-const std::string& CShaderPreset::GetShaderPreset() const
-{
-  return m_presetPath;
-}
-
-bool CShaderPreset::Update()
 {
   auto updateFailed = [this](const std::string& msg)
   {
     m_failedPaths.insert(m_presetPath);
-    CLog::Log(LOGWARNING, "CShaderPreset::Update: {}", msg);
+    CLog::Log(LOGWARNING, "CShaderPreset::SetShaderPreset: {}", msg);
```

Comment:
Have you thought about using CLog::LogF to automatically get standardized and full method signature logged?